src/lib/schematics/update/update.ts

```diff
@@ -78,38 +59,22 @@ export default function(): Rule {
         silent: false,
         ignoreErrors: true,
         tsConfigPath: tsconfig,
-      }), [downgradeTask]));
+      })));
     }
 
-    // Upgrade @angular/material back to 6.x.
-    const upgradeTask = context.addTask(new NodePackageInstallTask({
-      // TODO(mmalerba): Change "next" to ">=6 <7".
-      packageName: '@angular/cdk@next @angular/material@next'
-    }), allUpdateTasks);
-
     // Delete the temporary schematics directory.
-    context.addTask(new RunSchematicTask('ng-post-update', {
-      deletePath: schematicsTmpPath
-    }), [upgradeTask]);
-  };
-}
-
-/** Post-update schematic to be called when ng update is finished. */
-export function postUpdate(options: {deletePath: string}): Rule {
-  return (tree: Tree, context: SchematicContext) => {
-    tree.delete(options.deletePath);
-    context.addTask(new RunSchematicTask('ng-post-post-update', {}));
+    context.addTask(new RunSchematicTask('ng-post-update', {}), tslintFixTasks);
   };
 }
 
-/** Post-post-update schematic to be called when post-update is finished. */
-export function postPostUpdate(): Rule {
+/** Post-update schematic to be called when update is finished. */
+export function postUpdate(): Rule {
   return () => console.log(
       '\nComplete! Please check the output above for any issues that were detected but could not' +
       ' be automatically fixed.');
 }
 
-/** Gets the first tsconfig path from possibile locations based on the history of the CLI. */
+/** Gets the first tsconfig path from possible locations based on the history of the CLI. */
```

Comment:
Could you update this comment so that it says "Gets all tsconfig..." instead of the "Gets the first..."